### PR TITLE
Adds total summaries at top of status web page across all map instances 

### PR DESCRIPTION
**Pull request by tomballgithub**
## Description
Adds total summaries at top of status web page across all map instances whether in group-by-mainworker mode or not.  This was waiting on release of PR #1604 and basically adds up the stats for each main_worker and generates stats across all workers.  

These total stats include:  active workers, successes, failures, empties, skips, and captchas.

I had to muddle through learning quite a bit about javascript to make this work.  I am open to suggestions on how to clean things up.

The per hr and per month stats use an average of the elapsed time of each instance, assuming that they are launched at approximately the same time.  Any other method would be extravagant.  If instances are launched random times from each other the top-level per hr and per month numbers are not trustworthy.

## Motivation and Context
If you have multiple instances, there is not a way to see the totals across all instances.  PR #1604 added per-main-worker support and this builds upon that

## How Has This Been Tested?
Tested on my Windows server with 10 instances of 5-25 workers each
Verified totals and calculations are correct
Verified behavior works in group-by-mainworker mode and other mode

## Screenshots (if appropriate):
image
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by averylarry** [comment]
Is this really worth a database change?

**Comment by mew905** [comment]
this... this is beautiful and was the ultimate goal of 1587 but I don't have the skill to modify the status page. Thank you!

**Comment by BrownSlaughter** [comment]
Seems out of place, it would look better under the other text and start with"All instances:"

**Comment by tomballgithub** [comment]
@BrownSlaughter had good feedback so I updated the code and screenshots to make the information follow the style of the main workers

**Comment by tomballgithub** [comment]
Updated the status.js changes based on recent develop commit of style changes to this file.

**Comment by Zyrix** [comment]
From time to time I have a problem with this PR:

Traceback (most recent call last):
  File "runserver.py", line 91, in run
    run_old(*args, **kwargs)
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "pogom/search.py", line 475, in search_overseer_thread
    update_total_stats(threadStatus, last_account_status)
  File "pogom/search.py", line 541, in update_total_stats
    last_account_status[username] = copy.deepcopy(tstatus)
  File "/usr/lib/python2.7/copy.py", line 163, in deepcopy
    y = copier(x, memo)
  File "/usr/lib/python2.7/copy.py", line 256, in _deepcopy_dict
    for key, value in x.iteritems():
RuntimeError: dictionary changed size during iteration

I think it is related?

**Comment by tomballgithub** [comment]
The 'deepcopy' code being referenced in your error messaging clip is not unique to this PR.  It was in a previous commit to add captcha stats to the -PS screen.  @onilton needs to comment.

**Comment by tomballgithub** [comment]
This has been working flawlessly for me and is invaluable.  Anyone else want to review and give it a 2nd thumbs up?
image

**Comment by tomballgithub** [comment]
Updated this PR to show the # of instances being summed up as well as only show the summary at top if there is > 1 instance or if not in GroupByWorkers mode.  

For instance, this example shows 'Total Statistics across 15 instances'

image

File: pogom/models.py [hunk 1]
```diff
@@ -1110,6 +1110,13 @@ class MainWorker(BaseModel):
     accounts_working = IntegerField()
```
Comment (Alderon86) [inline pogom/models.py]: `SmallIntegerField()` for consistency

**Comment by tomballgithub** [comment]
I have been updating this since January.  I recommend that it needs to either be in or out @ 4.1.0.  The only tweaks I have been tracking would be 1) to only show this if the # of instances is > 1 and/or 2) have a configuration option to turn this on/off.  @sebastienvercammen

**Comment by tomballgithub** [comment]
Closing and replacing with #2372  after accidentally deleting the branch this was based on which means no further editing is possible.

**Pull request closed without merging**